sql-bom/build.gradle

```diff
@@ -42,7 +43,22 @@ dependencies {
         api "org.mariadb.jdbc:mariadb-java-client:$mariadbDriverVersion"
         api "mysql:mysql-connector-java:$mysqlDriverVersion"
 
+
         // jdbi
-        api "org.jdbi:jdbi3-core:$jdbiVersion"
+        platform("org.jdbi:jdbi3-bom:$jdbiVersion")
+
+        // Will remove these - just want to chat about it first
+        api "org.jdbi:jdbi3-core"
```

Comment:
Doesn't blow up, therefore the platform is being respected.

Reply:
The comments below should be removed. There will apparently be a difference: currently the `micronaut-sql` BOM would produce a constraint for `org.jdbi:jdbi3-oracle12:$jdbiVersion`. For the current version, which is `3.0.21`, as per https://github.com/micronaut-projects/micronaut-sql/blob/df34ee6e8e0b049517e38f103c13e3005a9e50e6/gradle.properties#L23 and confirmed by https://ge.micronaut.io/s/xfmwthgikr56s/dependencies?dependencies=jdbi&expandAll&focusedDependency=WzgsMSwxNDg0LFswLDAsWzFdXV0.

However the 3.0.21 BOM doesn't contain an entry for `oracle12`: https://search.maven.org/artifact/org.jdbi/jdbi3-bom/3.20.1/pom

So we need to keep the `api` constraint on `api "org.jdbi:jdbi3-oracle12:$jdbiVersion"` (the other modules are all present).

However, it seems that `jdbi3-oracle12` is _already_ broken since there's no such thing as `3.0.21`: https://repo1.maven.org/maven2/org/jdbi/jdbi3-oracle12/ (it seems to be versioned independently of the rest of the project). So either we need to use a specific version of this dependency, or we simply remove it from the BOM. @graemerocher wdyt?

Reply:
Was `jdbi3-oracle12` ever included here? Everything below is a comment that I added as a reference, not an old part of the BOM.

Reply:
Removed the comments to make this more clear.